src/build/index.js

```diff
@@ -91,9 +91,17 @@ export default function build(source, config = {}, typeDeps = [], infer = true)
     if (!Array.isArray(typeDeps)) {
       throw new Error('Can\'t use thunks as type dependencies for schema.');
     }
+
+    const schemaConfigKey = '__schema';
```

Comment:
nit: Can we have this constant be declared in the outermost scope with the name `SCHEMA_CONFIG_KEY`?